blog/2015/03/what-do-people-want-to-learn.html

```diff
@@ -0,0 +1,62 @@
+---
+layout: blog
+root: ../../..
+author: Tiffany Timbers
+title: "What Do People Want to Learn?"
+date: 2015-03-15
+time: "11:00:00"
+category: ["Assessment"]
+---
+<!-- start excerpt -->
+<p>
+  In the planning phase of organizing a Software Carpentry workshop
+  for my home department of Molecular Biology &amp; Biochemistry
+  I started to wonder what participants want to learn.
+  I designed a short survey to answer this question,
+  and from my small department,
+  ~20% (30 people) filled it in.
+  Here's what they said:
+</p>
+<!-- end excerpt -->
+<p>
+  <img src="timbers-q1.jpg" alt="Q1" />
+</p>
+<p>
+  <img src="timbers-q2.jpg" alt="Q2" />
+</p>
+<p>
+  <img src="timbers-q3.jpg" alt="Q3" />
+</p>
+<p>
+  (As with most good things in life,
+  the code for these plots can be <a href="https://github.com/ttimbers/2015-03-30_SFU/blob/master/Survey/MBB_SWC_Survey.R">found on Github</a>.)
+</p>
+<p>
+  I also asked survey participants to comment on any other computer programming and/or data science topics
+  they would be interested in learning about.
+  Here's what people said:
+</p>
+<ul>
+  <li>Circos plot!!!</li>
+  <li>VBA (any level)</li>
+  <li>Processing of strings as in sequencing data would be useful</li>
+  <li>Reproducible Research and experimental planning, ensuring that experiments are</li>
+  <li>well-designed, valid, reproducible and well-documented.</li>
```

Comment:
With bootstrap 3, it's good to add class="img-responsive" to images inside
posts so they are the right size no matter the window size (and the device
used to display the page).

On Sun, Mar 15, 2015 at 3:20 PM, W. Trevor King <notifications@github.com>
wrote:

> In blog/2015/03/what-do-people-want-to-learn.html
> <https://github.com/swcarpentry/site/pull/876#discussion_r26451028>:
>
> > +</p>
> > +<p>
> > +  (As with most good things in life,
> > +  the code for these plots can be <a href="https://github.com/ttimbers/2015-03-30_SFU/blob/master/Survey/MBB_SWC_Survey.R">found on Github</a>.)
> > +</p>
> > +<p>
> > +  I also asked survey participants to comment on any other computer programming and/or data science topics
> > +  they would be interested in learning about.
> > +  Here's what people said:
> > +</p>
> > +<ul>
> > +  <li>Circos plot!!!</li>
> > +  <li>VBA (any level)</li>
> > +  <li>Processing of strings as in sequencing data would be useful</li>
> > +  <li>Reproducible Research and experimental planning, ensuring that experiments are</li>
> > +  <li>well-designed, valid, reproducible and well-documented.</li>
>
> You probably dont' want the exta <li> here, since “well-designed…” is
> just the tail of the previous sentence.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/swcarpentry/site/pull/876/files#r26451028>.
>